src/pages/UpSertUser/InsertPage/index.tsx

```diff
@@ -0,0 +1,47 @@
+import { useHistory } from "react-router-dom";
+
+import useSnackbar from "src/components/SnackBars/useSnackbar";
+import { IUser } from "src/pages/Users/components/UserCard";
+
+import UpSertPage from "../UpSertPage";
+import useInsertUser from "../hooks/useInsertUser";
+
+const InsertPage = () => {
+    const history = useHistory();
+
+    const showSnackbar = useSnackbar();
+
+    const backToListPage = () => {
+        history.push("/user");
+    };
+
+    const showError = (message: string, type: "error" | "success") => {
+        showSnackbar({
+            color: type,
+            variant: "filled",
```

Comment:
I think we should make this the default variant for snackbar

Reply:
![image](https://user-images.githubusercontent.com/58205909/132014479-0e4ec350-5b9f-42ac-9487-a66f7615fa77.png)
I set filled is default variant for alert